projects/igniteui-angular/src/lib/grids/columns/column-group.component.ts

```diff
@@ -25,7 +25,7 @@ import { flatten } from '../../core/utils';
 export class IgxColumnGroupComponent extends IgxColumnComponent implements AfterContentInit {
 
     @ContentChildren(IgxColumnComponent, { read: IgxColumnComponent })
-    children = new QueryList<IgxColumnComponent>();
+    public children = new QueryList<IgxColumnComponent>();
```

Comment:
Should the list of children be `public` or `protected`?

Reply:
In this case the children field in `IgxColumnComponent` is declared as public field and we are not able to make it protected due to the extension, unless the parent one is changed too.

Reply:
The property is used by many components internally so it will cause many errors if we change the modifier.

Reply:
That property is used in many other compnents and for now we can't change the modifier. Otherwise plenty of errors will arise.